### PR TITLE
Changed helm repo URL to existing one.

### PR DESCRIPTION
**Pull request by karstengresch**
old: https://kubernetes-charts.storage.googleapis.com
new: https://charts.helm.sh/stable

### CODE REVIEW

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* User @karstengresch isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

_Regards,
CLA GitHub bot_

**Comment by scbizu** [comment]
Thanks you for the PR , please ensure to make the CLA happy .

**Comment by antoinedeschenes** [comment]
The chart is now hosted there in fact: https://chartmuseum.github.io/charts

File: README.md [hunk 1]
```diff
@@ -497,7 +497,7 @@ You can also view it on [Helm Hub](https://hub.helm.sh/charts/stable/chartmuseum
 
 To install:
 ```bash
-helm repo add stable https://kubernetes-charts.storage.googleapis.com
+helm repo add stable https://charts.helm.sh/stable
```
Comment (jdolitsky) [inline README.md:500]: @antoinedeschenes is correct. Can we switch this to
`helm repo add chartmuseum https://chartmuseum.github.io/charts` ?

**Review by jdolitsky — CHANGES_REQUESTED** [review]
@antoinedeschenes

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**